### PR TITLE
Hi, so what is this project about?

### PR DESCRIPTION
**Pull request by ts-vadim**
I saw your invitation to this project but I don't have almost any idea on what do you atually want to create. I mean what kind of a VPN tool this is going to be? 

By the way I made this pull request because I couldn't find your email

### CODE REVIEW

**Comment by Rayan25062011** [comment]
> 

Well, short answer: I needed people to help maintain this project and try to make it available to ‘pip’ since i have no idea how. Thats if you want to do it.

**Review by Rayan25062011 — APPROVED** [review]
Thanks a lot! Didn’t see that, good job.

**Comment by ts-vadim** [comment]
TLDR: I want to help but I don't know how and with what exactly.

So as I can understand this is going to be a python library (not a cli tool? or both?) 
which would pass internet traffic from the script itself or OS (?) through a VPN server which it would previously connect to? 
I mean, will the script use `requests.whatever(proxies=...)`, OS VPN manager or something else? What the libraries you're going to to use?

Btw I just edited 1 line in `LICENSE` for fun but thanks)

Sorry if I bother you but I also want to make similar app 
that would be efficient and easy to use (and this topic is actually relevant in my country) 
but I still can't get the image of your idea (and the final program, what it would look like).
I think it would be helpful to have some kind of TDD (especially for new contributors)

**Comment by Rayan25062011** [comment]
Well, you can help by adding some features and maintain it if you’d like? 
And you got everything right except it just goes throughout the script and not the OS.

And its a CLI and a python library. If you can’t really get an image of my idea, well its just a simple script to hide the clients IP address by changing the way the system receives it.
If you’d like to see the output you can run using the instructions or just see the output section in the readme.md

**Review by Rayan25062011 — APPROVED** [review]
Good job! Keep going
